src/swh_api_functions.py

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# heritedcode is a trademark of nexB Inc.
+# SPDX-License-Identifier: Apache-2.0
+# See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+# See https://github.com/nexB/heritedcode for support or download.
+# See https://aboutcode.org for more information about nexB OSS projects.
+#
+
+
+import requests
+
+base_swh_api_url = "https://archive.softwareheritage.org/api/1/"
```

Comment:
This will be a constant, so I think we should write it as `BASE_SWH_API_URL`

Reply:
Okay